src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -303,23 +303,19 @@ type private DiffixSum() =
   let increaseUnaccountedFor valueIncrease i =
     let absValueIncrease = abs (valueIncrease)
 
-    if valueIncrease > 0.0 then
-      state.Positive.[i].UnaccountedFor <- state.Positive.[i].UnaccountedFor + absValueIncrease
-    else if valueIncrease < 0.0 then
-      state.Negative.[i].UnaccountedFor <- state.Negative.[i].UnaccountedFor + absValueIncrease
-    else
+    if valueIncrease >= 0.0 then
       state.Positive.[i].UnaccountedFor <- state.Positive.[i].UnaccountedFor + absValueIncrease
```

Comment:
Isn't `>=` redundant because only the `>` case has an effect?

Reply:
yeah, for the unaccounted for that's indeed true. I wanted to keep it parallel to the `increaseContribution` form, but let's do `>` / `<`